package com.thinkinginjava.error_handling_with_exceptions.example.chapter12_11;

/**
 * page 275
 *   -- chapter_12 通过异常处理错误
 *
 *   12.11 异常匹配
 */